installer/conf/omsagent.d/npmd.conf

```diff
@@ -0,0 +1,19 @@
+<source>
+    type npmd
+    tag oms.npmd
+    location_unix_endpoint  "/var/opt/microsoft/omsagent/run/npmdagent.sock"
+    location_control_data   "/etc/opt/microsoft/omsagent/conf/npmd_agent_config.xml"
+    location_agent_binary   "/opt/microsoft/omsagent/npmd_agent"
+    location_uuid_file      "/etc/opt/microsoft/omsagent/conf/npmd_agent_guid.txt"
+</source>
+
+<match npmd.oms.blob>
```

Comment:
Yes we agree that it can be handled by default output plugin. We are basically asking if we can have this so as to have more control over the event transmission. Similar to how "oms_heartbeat" and "monitor_agent" have redefined this in "monitor.conf"